app/mailers/application_mailer.rb

```diff
@@ -0,0 +1,4 @@
+class ApplicationMailer < ActionMailer::Base
```

Comment:
I don't think this is used. Do we need it?

Reply:
Do we need to remove from gov template as well as from register?